project1/report.Rmd

```diff
@@ -76,5 +76,52 @@ data[, serum_cat := cut(serum_bilirubin, c(0, 1.1, 3.3, max(serum_bilirubin)))]
 # Code
 
 ```{r ref.label = knitr::all_labels(), echo = TRUE, eval = FALSE}

```

Comment:
This code chunk is special in RMarkdown in that when we "knit" this .Rmd file, it takes all of the code from the previous chunks and pastes them here in the .pdf, without evaluating or running the code again. Could you paste this code in its own code chunk above this one? That way when you "knit" a `.pdf` of this report will be produced, and we'll be able to see the plots, summaries, and figures. Right now the code is in a chunk where `eval=FALSE` so none of the code has been run when knit.